lib/admin.php

```diff
@@ -66,104 +70,110 @@ function wc_admin_register_page( $options ) {
 function wc_admin_register_pages() {
 	global $menu, $submenu;
 
-	add_submenu_page(
-		'woocommerce',
-		__( 'WooCommerce Dashboard', 'wc-admin' ),
-		__( 'Dashboard', 'wc-admin' ),
-		'manage_options',
-		'wc-admin',
-		'wc_admin_page'
-	);
+	if ( wc_admin_is_feature_enabled( 'dashboard' ) ) {
+		add_submenu_page(
+			'woocommerce',
+			__( 'WooCommerce Dashboard', 'wc-admin' ),
+			__( 'Dashboard', 'wc-admin' ),
+			'manage_options',
+			'wc-admin',
+			'wc_admin_page'
+		);
+	}
 
-	add_menu_page(
-		__( 'WooCommerce Analytics', 'wc-admin' ),
-		__( 'Analytics', 'wc-admin' ),
-		'manage_options',
-		'wc-admin#/analytics/revenue',
-		'wc_admin_page',
-		'dashicons-chart-bar',
-		56 // After WooCommerce & Product menu items.
-	);
+	if ( wc_admin_is_feature_enabled( 'analytics' ) ) {
```

Comment:
> I'm thinking it's mostly okay

I think so too, and wrapping the endpoints in different checks just might be confusing/more trouble than it is worth. For example, dashboard relies on the same endpoints as analytics, and I think there is some overlap between the activity panel too (adding additional fields to a response for example).

> but if they're disabling because of a conflict in WC core where our API files override, this might cause an issue.

Since these are versioned from the core endpoints I think users shouldn't notice an issue, unless they are testing `plugin`/`development`, (i.e these are v4 and anything "real" will use v3). As long as we keep bumping the `namespace` in `wc-admin` whenever we do a big core merge we should be OK -- So when core hits v4, we should use v5.